js/views/chatAppVw.js

```diff
@@ -0,0 +1,155 @@
+var __ = require('underscore'),
+    Backbone = require('backbone'),
+    $ = require('jquery'),
+    chatCollection = require('../collections/chatCl'),
+    chatView = require('./chatVw'),
+    chat = require('../models/chatMd'),
+    loadTemplate = require('../utils/loadTemplate');
+Backbone.$ = $;
+
+module.exports = Backbone.View.extend({
+
+  el: '#sideBar',
+
+  events: {
+    'click .js-newChat': 'newChat',
+    'click .js-closeChat': 'closeChat',
+    'click .js-closeConversation': 'closeConversation'
+  },
+
+  initialize: function(options){
+    var self = this;
+    this.options = options || {};
+    this.parentEl = $(options.parentEl);
+    this.slidOut = false;
+
+
+    // Render chat list items
+    this.listWrapper = $('<div class="border0 custCol-border-secondary flexRow"></div>');
+    this.chats = new chatCollection();
+    this.chats.url = options.model.get('serverUrl') + "get_chat_messages?guid=" + options.model.get('guid');
+    this.chats.fetch({
+      success: function(chats, response) {
+        if(chats.models.length < 1) {
+          self.renderNoneFound();
+        } else {
+          __.each(chats.models, function (chat) {
+            "use strict";
+            chat.set('avatarURL', self.options.serverUrl + "get_image?hash=" + chat.get('image_hash') + "&guid=" + chat.get('guid'));
+            self.renderChat(chat);
+          });
+          self.parentEl.html(self.listWrapper);
```

Comment:
Is there a timing issue here? Depending on how long the fetch takes, the listWrapper could be added to the parentEl before or after the rest of the view renders. It might be better to move placing the listWrapper into the callback of loadTemplate.